### PR TITLE
genesis update #1197

### PR DESCRIPTION
**Pull request by r1di**
they updated their api and obfuscated alot..so no more chapter names, they hide it behind obfuscated  functions , its working now at least...

### CODE REVIEW

**Comment by r1di** [comment]
fixes #1197

File: scripts/multisrc/madara/sources.json
Comment (error7404) [inline scripts/multisrc/madara/sources.json:1]: You forgot about version increase

**Comment by 1-minute-to-midnight** [comment]
Titles don't seem to be working

**Comment by Palloxin** [comment]
Screenshot_20241005-135205_1
"Murim crazy bitch"

**Comment by r1di** [comment]
yes, because it obfuscated(encrypted), this was intended. But someone on discord got it fixed, he has just to merge his code, then we get back titles etc

**Pull request merged**